src/Api/getAllAssignmentSettings.php

```diff
@@ -74,16 +74,15 @@
           "proctorMakesAvailable" => $row['proctorMakesAvailable'] == '1' ? true : false,
           "doenetId" => $row['doenetId']
     );
+  } else {
+    http_response_code(404);
```

Comment:
@djnykamp is this what you wanted to be removed? Not sure why its back but want to review it with you